src/containers/Game/BattleArena.js

```diff
@@ -47,53 +47,57 @@ timerAttackType = (x, y) => {
 
   render() {
     return (
-      <div className={this.props.background}>
-        <HealthBar side='left' percentage={this.calulatePercentage(0)}/>
-        <HealthBar side='right' percentage={this.calulatePercentage(0)}/>
-        {this.state.playerStatus === 'idle' &&
-        <div>
-          <Player side='left' character={this.props.player1} imgSrc={require (`../../img/classes/${this.props.player1}-${this.state.playerStatus}.gif`)} status={this.state.playerStatus} attackType={this.state.attackType}/>
-          <Player side='right'character={this.props.player2} imgSrc={require (`../../img/classes/${this.props.player2}-${this.state.playerStatus}.gif`)} status={this.state.playerStatus} attackType={this.state.attackType}/>
-          {this.timerPlayerState('move', 450)}
+      <div>
+      {this.props.player1 !== 'no' && this.props.player !== 'no' &&
```

Comment:
Why does this use player1 and player when the rest uses player1 & player2?

Reply:
forgot to add 2 fixed it just now.